apps/tenant-management-webapp/src/app/pages/admin/services/configuration/revisions/revisionsTable.tsx

```diff
@@ -165,6 +165,13 @@ const RevisionTableComponent: FunctionComponent<RevisionTableComponentProps> = (
       dispatch(getConfigurationActive(service));
     }
   }, [indicator, revisions]);
+  if (
+    !configurationRevisions[service]?.revisions.next &&
+    !configurationRevisions[service]?.revisions?.latest &&
+    configurationRevisions[service]?.revisions.result.length
```

Comment:
It better configurationRevisions[service]?.revisions?.result?.length to avoid future bug.

Reply:
updated change